readthedocs/projects/forms.py

```diff
@@ -211,6 +212,59 @@ class Meta(object):
             # 'num_major', 'num_minor', 'num_point',
         )
 
+    def __init__(self, *args, **kwargs):
+        super(ProjectAdvancedForm, self).__init__(*args, **kwargs)
+
+        # default_version ChoiceField
+        self.versions_qs = self.instance.versions.all()
+        self.active = self.versions_qs.filter(active=True)
```

Comment:
Why do you use `self.` to declare these variables? Shouldn't they just be local variables?

Reply:
Right, I got it